zebra-network/src/meta_addr/tests/prop.rs

```diff
@@ -15,8 +15,10 @@ use tokio::{runtime, time::Instant};
 use tower::service_fn;
 use tracing::Span;
 
+use zebra_chain::serialization::{DateTime32, ZcashDeserialize, ZcashSerialize};
```

Comment:
Oops, should have used a compiler:
```suggestion
use zebra_chain::serialization::DateTime32;
```